### PR TITLE
[MeshingApplication] Improving EnsureModelPartOwnsProperties

### PR DESCRIPTION
**Pull request by loumalouomega**
Adding RecursiveEnsureModelPartOwnsProperties and adding option to clear first all properties

### CODE REVIEW

File: applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp [hunk 1]
```diff
@@ -47,11 +47,40 @@ namespace Python
...
+{
+    MeshingUtilities::RecursiveEnsureModelPartOwnsProperties(rModelPart, RemovePreviousProperties);
+}
+
+void EnsureModelPartOwnsProperties(ModelPart& rModelPart)
+{
+    MeshingUtilities::EnsureModelPartOwnsProperties(rModelPart);
+}
+
+void EnsureModelPartOwnsPropertiesWithClear(
```
Comment (philbucher) [inline applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp:68]: I was discussing this week with jordi, maybe in the future we should start using fct pointers, would clean up a lot

Comment (loumalouomega) [reply applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp:68]: fct pointers¿?

Comment (philbucher) [reply applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp:68]: E.g. like this: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python/add_data_communicator_to_python.cpp#L77
Then we don’t need the aux function

**Review by philbucher — APPROVED** [review]
btw duplicate of #5583 ?

**Comment by loumalouomega** [comment]
Yes, I duplicated, my bad

**Pull request merged**